build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -55,3 +40,42 @@ apiServer:
 
   # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
   #tlsMinVersion:
+
+# flowCollectorSink contains external IPFIX or JSON collector related configuration options.
+flowCollectorSink:
```

Comment:
maybe we should add an `enable` switch here as well for consistency?

Reply:
added.